### PR TITLE
Trim description to 80, and end with '[...]'

### PR DESCRIPTION
**Pull request by allenh1**
This cleans up the truncation a little better.

connects to #35

### CODE REVIEW

File: superflore/utils.py [hunk 1]
```diff
@@ -63,6 +63,17 @@ def sanitize_string(string, illegal_chars):
     return ret
 
 
+def trim_string(string, length=80):
+    if len(string) < length:
+        return string
+    end_string = '[...]'
+
+    return '{0}{1}'.format(
+        string[:length - len(end_string)],
```
Comment (tfoote) [inline superflore/utils.py]: You don't need to bother with the format here. You can just add the strings. 

`return string[:length - len(end_string)] + end_string`

File: superflore/utils.py [hunk 2]
```diff
@@ -63,6 +63,15 @@ def sanitize_string(string, illegal_chars):
     return ret
 
 
+def trim_string(string, length=80):
+    if len(string) < length:
+        return string
+    end_string = '[...]'
+
+    return string[:length - len(end_string)]\
```
Comment (tfoote) [inline superflore/utils.py]: Don't wrap here, it's only 57 characters total. One line is much more readable.

Comment (allenh1) [inline superflore/utils.py]: Got it -- fixed.

**Comment by Alessandro-Barbieri** [comment]
thanks

**Comment by allenh1** [comment]
Sure thing!

**Pull request merged**